docs-md/tutorials/basics-docker.md

```diff
@@ -252,70 +294,153 @@ Optional: Show all streams and tables.
+   |ROWTIME             |ROWKEY        |REGISTERTIME  |GENDER   |REGIONID  |USERID       |
+   +--------------------+--------------+--------------+---------+----------+-------------+
+   |1581077558655       |User_9        |1513529638461 |OTHER    |Region_1  |User_9       |
+   |1581077561454       |User_7        |1489408314958 |OTHER    |Region_2  |User_7       |
+   |1581077561654       |User_3        |1511291005264 |MALE     |Region_2  |User_3       |
+   |1581077561857       |User_4        |1496797956753 |OTHER    |Region_1  |User_4       |
+   |1581077562858       |User_8        |1489169082491 |FEMALE   |Region_8  |User_8       |
+   Limit Reached
+   Query terminated
+   ```
+   !!! note
+         Push queries on tables will output the full history of the table that is stored
+         in the Kafka changelog topic, i.e. it will output historic data, followed by the
+         stream of updates to the table. It is therefore likely that rows with matching
+         `ROWKEY` are output as existing rows in the table are updated.
+
+2. View the data in your pageviews_original stream by issuing the following
+   push query:
+   ```sql
+   SELECT viewtime, userid, pageid FROM pageviews_original emit changes LIMIT 3;
+   ```
+
+   Your output should resemble:
+
+   ```
+   +--------------+--------------+--------------+
+   |VIEWTIME      |USERID        |PAGEID        |
+   +--------------+--------------+--------------+
+   |1581078296791 |User_1        |Page_54       |
+   |1581078297792 |User_8        |Page_93       |
+   |1581078298792 |User_6        |Page_26       |
+   Limit Reached
+   Query terminated
+   ```
+
+   !!! note
+      By default, push queries on streams only output changes that occur
+      after the query is started, i.e. historic data is not included.
+      Run `set 'auto.offset.reset'='earliest';` to update your session
+      properties if you want to see the historic data.
+
 Write Queries
 -------------
 
 These examples write queries using ksqlDB.
 
-!!! note
-	By default ksqlDB reads the topics for streams and tables from
-    the latest offset.
-
 ### 1. Create a query that returns data from a ksqlDB stream
 
-Use `SELECT` to create a query that returns data from a STREAM. This
-query includes the `LIMIT` keyword to limit the number of rows
-returned in the query result. Note that exact data output may vary
-because of the randomness of the data generation.
+Create query that enriches the pageviews data with the user's gender
+and regionid from the users table. The following query enriches the
+`pageviews_original` STREAM by doing a `LEFT JOIN` with the
+`users_original` TABLE on the userid column.
 
 ```sql
-SELECT pageid FROM pageviews_original LIMIT 3;
+SELECT users_original.userid AS userid, pageid, regionid, gender
+  FROM pageviews_original
+  LEFT JOIN users_original
+    ON pageviews_original.userid = users_original.userid
+  EMIT CHANGES
+  LIMIT 5;
```

Comment:
Switched the example to something more useful to demo how a user might first get the query working with a transient query, before then converting it to a persistent query in the next step.